crates/iota/src/client_ptb/parser.rs

```diff
@@ -268,11 +268,11 @@ impl<'a, I: Iterator<Item = &'a str>> ProgramParser<'a, I> {
     /// Parse a transfer-objects command.
     /// The expected format is: `--transfer-objects [<from>, ...] <to>`
     fn parse_transfer_objects(&mut self) -> PTBResult<Spanned<ParsedPTBCommand>> {
-        let transfer_froms = self.parse_array()?;
+        let transfer_from = self.parse_array()?;
         let transfer_to = self.parse_argument()?;
-        let sp = transfer_to.span.widen(transfer_froms.span);
+        let sp = transfer_to.span.widen(transfer_from.span);
         Ok(sp.wrap(ParsedPTBCommand::TransferObjects(
-            transfer_froms,
+            transfer_from,
```

Comment:
I kinda prefer the plural forms here to be honest, it's weird sure but makes sense and easier to understand

Reply:
is it even multiple things? it's not clear

Reply:
Yes these are vectors

Reply:
Then they should both be either `transfers_x` or as-is

Reply:
Sure, but does it really matter? I mean is it so important that this is plural that we allow a typo across the while workspace?

Reply:
I'd say so yes, this code is complex enough as it is, no need to make it even more by having wrong names

Reply:
But is it really wrong? It's arbitrary

Reply:
Yes it is really wrong. Variable names need to reflect the singularity/plurality of what they describes, I don't think this is so crazy nor arbitrary 🤷🏻‍♂️

Reply:
Well I totally disagree 🤷‍♀️

Reply:
I think I would prefer the plural form, but maybe add more semantics? It is transfer from what? transfer_from_objs, transfer_from_addrs? (I don't know exactly what this code is doing.)

Reply:
Please rename `transfer_froms` to `objects` and `transfer_to` to `to`

Reply:
done